src/YQReplacePoint.cc

```diff
@@ -42,10 +42,21 @@ void YQReplacePoint::showChild()
 {
     YWidget * child = firstChild();
 
+    /* This needs to be done so the UI won't crash if the are no children. */
     if ( child )
     {
-	QWidget * qChild = (QWidget *) child->widgetRep();
-	qChild->show();
+        QWidget * qChild = (QWidget *) child->widgetRep();
+        qChild->show();
+
+        /* Refresh the whole UI so it doesn't get stalled in some situations. */
```

Comment:
great, much better now, just one final note, can you add also reference to github issue, so we have in future place where we can see some discussion about this issue? Is there such issue?